pages/common/nasm.md

```diff
@@ -0,0 +1,23 @@
+# nasm
+
+> The Netwide Assembler, a portable 80x86 assembler.
+
+- Assemble raw binary file `source` from `source.asm`:
+
+`nasm {{source.asm}}`
+
+- Assemble and generate an assembly listing file:
```

Comment:
You're perfectly correct @agnivade, nasm assembles a binary file from the instructions in `source.asm` and additionally creates a listing file. Should I make this clearer in the example text?